drivers/lte_link_control/Kconfig

```diff
@@ -91,6 +106,16 @@ config LTE_PDP_CONTEXT
 		The +CGDCONT command defines Packet Data Protocol (PDP) Context.
 		For reference, see 3GPP 27.007 Ch. 10.1.1
 
+config LTE_PDN_AUTH_CMD
+	bool "Enable PDN connection authentication AT command"
+
+config LTE_PDN_AUTH
+	string "PDN connection authentication parameters"
+	depends on LTE_PDN_AUTH_CMD
```

Comment:
For `LTE_LOCK_PLMN_STRING` you have added an if condition on `LTE_LOCK_PLMN`.
Here you have used `depends on`. To keep it consistent, I suggest to use one or the other in this file.
Overall LGTM, @joakimtoe since you had a look, could please approve if it's alright?

Reply:
As you can see that I copied "config LTE_PDP_CONTEXT". Yes it'd be good to us _if_. Actually CMAKE gives a warning of "unused" on this. Should I change for both?

Reply:
Thanks for the comment. I just committed again for your review.